dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task220402UpdateDateTimezones.java

```diff
@@ -66,6 +66,22 @@ public boolean forceRun() {
         return DbConnectionFactory.isMsSql();
     }
 
+    @Override
+    public void executeUpgrade() throws DotDataException, DotRuntimeException {
+        tablesCount = 0;
```

Comment:
Isn't this conflicting with class attribute with same name?

Reply:
The variable visibility in the parent class was changed to `protected`, so we're good.